bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -1588,6 +1588,21 @@
         expect(file.deleted?).to eq true
       end
 
+      it "does not delete cached files marked as persistent" do
+        # This is not officially supported and may be removed without notice.
+        ::Bundler.settings.temporary(persistent_gems_after_clean: ["business"])
```

Comment:
Should we try adding this config from the `Gemfile` using a file that's required? Unsure if bundler reads and sets this for us.

Reply:
Yeah good shout